src/components/pages/TodoPage.js

```diff
@@ -24,9 +24,28 @@ export default class TodoPage extends Component {
                 Completed
               </th>
             </tr>
+            <tr>
+              <th>
```

Comment:
look closely at the table examples

everything in the head should be inside a ```<thead>```
everything in the body should be inside a ```<tbody>```

in the head, each piece of data is wrapped in a ```<th>```
in the body, each piece of data is wrapped in a ```<td>```